Python/pywarpx/Particles.py

```diff
@@ -1,12 +1,13 @@
 # Copyright 2017-2020 Andrew Myers, David Grote
+# 2022 Ryan Sandberg
 #
 # This file is part of WarpX.
 #
 # License: BSD-3-Clause-LBNL
 
 from .Bucket import Bucket
 
-particles = Bucket('particles', species_names=[], rigid_injected_species=[])
+particles = Bucket('particles', species_names=[], photon_species=[],rigid_injected_species=[])
```

Comment:
WarpX requires the line `particles.photon_species = photons` and this seemed like an easy way to be sure that line gets into the input deck. I only added photons in the first place because the reduced diagnostics test has photons.

Reply:
Ok, this is good. Though see my comment regarding setting up the species. And add a space after the comma.